config/500-webhook.yaml

```diff
@@ -15,10 +15,56 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: webhook
+  name: eventing-webhook
   namespace: knative-eventing
 spec:
   replicas: 1
+  selector:
+    matchLabels: &labels
+      app: eventing-webhook
+      role: eventing-webhook
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+      labels: *labels
+    spec:
+      serviceAccountName: eventing-webhook
+      containers:
+      - name: eventing-webhook
+        terminationMessagePolicy: FallbackToLogsOnError
+        # This is the Go import path for the binary that is containerized
+        # and substituted here.
+        image: github.com/knative/eventing/cmd/webhook
+        volumeMounts:
+        - name: config-logging
+          mountPath: /etc/config-logging
+        env:
+        - name: SYSTEM_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: CONFIG_LOGGING_NAME
+          value: config-logging
+        - name: WEBHOOK_NAME
+          value: eventing-webhook
+        resources:
+          limits:
+            memory: 1000Mi   # An initial guess, but consistent with serving
+      volumes:
+        - name: config-logging
+          configMap:
+            name: config-logging
+
+---
+
```

Comment:
Add a TODO to remove in 0.8.

Reply:
Will there be any conflicts between the two running simultaneously?

Reply:
The two won't run simultaneously (at least for more than a moment), because this YAML sets replicas=0 on the old Deployment, which will probably take effect around the same time as the new Deployment is started (probably faster, but not taking a bet).

If both end up running at once, they will both register their mutating webhook, but that should be fine -- whichever one is called earlier will do the defaulting, and the other will simply validate the same data which has been validated before.